packages/frontend/src/retro/components/buttons/ImportRetroMenuItem.tsx

```diff
@@ -29,6 +29,7 @@ export function ImportRetroMenuItem() {
       participants: {},
       waitingList: {},
       isVotingEnabled: false,
+      cardVotingLimit: Number.MAX_VALUE,
```

Comment:
By default it's 1. We should extract this initial config into a global variable and reuse it here maybe...

```suggestion
      cardVotingLimit: 1,
```